consensus/consensus-types/src/common.rs

```diff
@@ -1,58 +1,72 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::proof_of_store::ProofOfStore;
 use aptos_crypto::HashValue;
+use aptos_types::validator_verifier::ValidatorVerifier;
 use aptos_types::{account_address::AccountAddress, transaction::SignedTransaction};
 use rayon::prelude::*;
 use serde::{Deserialize, Serialize};
-use std::{fmt, fmt::Write};
+use std::collections::HashSet;
+use std::fmt;
+use std::fmt::Write;
 
 /// The round of a block is a consensus-internal counter, which starts with 0 and increases
 /// monotonically. It is used for the protocol safety and liveness (please see the detailed
 /// protocol description).
 pub type Round = u64;
 /// Author refers to the author's account address
 pub type Author = AccountAddress;
-
 #[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
 pub struct TransactionSummary {
     pub sender: AccountAddress,
     pub sequence_number: u64,
 }
-
 impl fmt::Display for TransactionSummary {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(f, "{}:{}", self.sender, self.sequence_number,)
     }
 }
-
 #[derive(Clone)]
 pub struct RejectedTransactionSummary {
     pub sender: AccountAddress,
     pub sequence_number: u64,
     pub hash: HashValue,
 }
-
 /// The payload in block.
 #[derive(Deserialize, Serialize, Clone, Debug, PartialEq, Eq)]
 pub enum Payload {
+    Empty,
```

Comment:
we can only append entries to the end of an enum to maintain backwards compatibility @bchocho

Reply:
Ah thanks! Pretty simple change :) @sasha8

Reply:
Thanks.

Reply:
@sasha8 looks like this isn't resolved. `Empty` needs to be moved to after `DirectMempool` for compat tests to succeed